autoload/db_ui/table_helpers.vim

```diff
@@ -32,6 +32,80 @@ let s:mysql = {
       \ 'Primary Keys': "SELECT * FROM INFORMATION_SCHEMA.TABLE_CONSTRAINTS WHERE TABLE_SCHEMA = '{schema}' AND TABLE_NAME = '{table}' AND CONSTRAINT_TYPE = 'PRIMARY KEY'",
       \ }
 
+let s:oracle_from = join(
+      \   [
+          \ 'FROM all_constraints N',
+          \ 'JOIN all_cons_columns L',
+          \   "\tON N.constraint_name = L.constraint_name",
```

Comment:
Here, I tested to ensure that `\t` will respond to the users `&expandtab` option.

It just looked odd having an enormous statement on one line (as you said), so I made it more compact according to what I've seen from other devs in my shop.

Reply:
These built in helpers are not really formatted, at least not for other db types. 99% of the time people just need to see the end result (columns, indexes, etc.), they don't really care about the query. It can also be easily formatted with some tools (`coc-sql` if you use coc, or `pg_format` external tool). So let's not bother too much to format it to be readable, just to keep it consistent with other db types. Narrowing this down to a single line without the array and join should be sufficient. 
I just wanted to avoid long strings in the objects that can cause some confusion.

Reply:
What about [`s:basic_constraint_query`](https://github.com/kristijanhusak/vim-dadbod-ui/blob/master/autoload/db_ui/table_helpers.vim#L1-L10) and the [sqlserver](https://github.com/kristijanhusak/vim-dadbod-ui/blob/master/autoload/db_ui/table_helpers.vim#L43-L91) queries?

Should I get rid of all formatting or make them minimally formatted, like those?

Reply:
Yeah something like that will work.